airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/utils.py

```diff
@@ -205,42 +205,56 @@ def transform_targeting_criteria(
             }
 
     """
+
+    def unnest_dict(nested_dict: Dict) -> Iterable[List]:
+        """
+        Unnest the nested dict to simplify the normalization
+
+        EXAMPLE OUTPUT:
+            [
+                {"type": "some_key", "values": "some_values"},
+                ...,
+                {"type": "some_other_key", "values": "some_othre_values"}
+            ]
+        """
+
+        result = []
+        for key, value in nested_dict.items():
+            values = []
+            if isinstance(value, List):
+                if isinstance(value[0], str):
```

Comment:
please handle a case when `value` is dict or list and is empty.

Reply:
We have such checks:
```
...
elif isinstance(value[0], Dict):
    for v in value:
        values.append(v)
elif isinstance(value, Dict):
    values.append(value)
...
```

Reply:
working with `value[0]` will be able to raise an exception if `value` is empty.
Could you add 2 test cases for checking of this situations [here](https://github.com/airbytehq/airbyte/pull/6610/files#diff-04ac0ab8c476dd546da935fca4bbba772a6bd1184998ec6cc829e8b792e7433eR54) ?
```
 "custom_empty_obj_var": {},
 "custom_empty_arr_var": [],
```